Airports/EGXW/Positions.txt

```diff
@@ -1,3 +1,3 @@
-EGXW_APP:Waddington Approach:119.500:XWR:R:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
+EGXW_APP:Waddington Approach:128.955:XWR:R:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
 EGXW_P_APP:Waddington Talkdown:123.300:XWP:P:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
```

Comment:
```suggestion
EGXW_APP:Waddington Approach:128.955:XWR:R:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
EGXW_L_APP:Waddington Zone:119.505:XWL:R:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
EGXW_P_APP:Waddington Talkdown:123.300:XWP:P:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
```
So as I understand it, we're effectively gaining a position since they're adding a VHF frequency for the APP position, so we need to change the APP freq and add a LARS position definition, is that what you mean @Robbo599? Something like this?

@AliceFord thoughts? I don't think the issue was 100% clear.

Reply:
Given I'm keen to release the SF punctually this cycle (AIRAC day is Boxing Day), I'll commit this change now and request re-review.